africa/cape_verde/cape_verde.md

```diff
@@ -0,0 +1,5 @@
+# Cape Verde
+
+It's a beautiful, small isolated island. Close to the ocean.
+
+https://en.wikipedia.org/wiki/Cape_Verde
```

Comment:
Links should be hyperlinked, i.e., with the syntax `[words explaining the link](https:......)`.